pom.xml

```diff
@@ -59,7 +59,7 @@
         <scm.branch>master</scm.branch>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <groovy.version>4.0.1</groovy.version>
-        <groovy.range>[3.0,4.0)</groovy.range>
+        <groovy.range>[3.0,5.0)</groovy.range>
```

Comment:
Wouldn't it be better to have range [4.0,5.0) ?

Reply:
@ponziani Indeed, my mistake! I changed the version after this commit.